docs/index.rst

```diff
@@ -88,3 +88,4 @@ Contents
    bugs.rst
    contributing.rst
    frequently-asked-questions.rst
+   lll.rst
```

Comment:
Do we really want to add another high level entry to the toc?

Reply:
personally i would prefere to find this info right where the question "What is this LLL thing???" raises (within the code). The text is fine, it clarifies everything.

Reply:
> Do we really want to add another high level entry to the toc?
 
Yes, that is the easy way to find it. I hope we remove all of LLL with 0.6.0.